src/index.ts

```diff
@@ -1,7 +1,19 @@
 import { Pool, PoolClient } from 'pg'
 import { Migrator } from './migrator'
+
 const UniqueIndexViolationErrCode = '23505'
-const pool = new Pool({ connectionString: process.env.DATABASE_URL })
+
+const DefaultTimeout = 20_000
+const DefaultPoolSize = 10
+
+const pool = new Pool({
+  connectionString: process.env.DATABASE_URL,
+  connectionTimeoutMillis: Number(process.env.DATABASE_CONN_TIMEOUT ?? DefaultTimeout),
+  query_timeout: Number(process.env.DATABASE_QUERY_TIMEOUT ?? DefaultTimeout),
```

Comment:
is kebab case correct here? I know PG is using it, but is the node driver passing through or camelizing it?

Reply:
default timeout shouldn't be the default for both dialing a connection and executing a query, the latter is usually orders of magnitude longer.

Reply:
>is kebab case correct here

Yes it's kebab here (actually it's snake case), for some reason they have such inconsistency in param names, they probably left original params unchanged in name but those which they somehow mapping they named in camelCase.

>default timeout shouldn't be the default for both dialing a connection and executing a query, the latter is usually orders of magnitude longer.

Yeah I know, decided to keep it simple here, but you're right, fixed.